### PR TITLE
Added czech language

### PR DESCRIPTION
**Pull request by ondraczsk**
not stable, needs fix links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +60,59 @@ Developing/modifying this project is easy.<br>
 We are still rewriting the base to make sure ClearSky has the best developing feel.<br>
 ClearSky is not just for a CLEAN feel for users - it's also for developers!<br>
 
+# <a name="CZE"></a>Česky
```
Comment (robske110) [inline README.md:63]: seems to acually work

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿## Choose Language:   
+﻿## Choose Language: 
+####  [Česky/Czech](#CZE)
```
Comment (robske110) [inline README.md:2]: please do not put this all the way up

**Review by robske110 — CHANGES_REQUESTED** [review]
Please check the following stuff:

**Comment by inxomnyaa** [comment]
updates?

**Comment by inxomnyaa** [comment]
Changes not applied

**Pull request closed without merging**